docs/evaluation/cache.rst

```diff
@@ -122,26 +168,23 @@ the `startup` and `teardown` methods are called automatically by
     >>> print(result.format())
     Method                = levmar
     Statistic             = chi2gehrels
-    Initial fit statistic = 2.4176
-    Final fit statistic   = 0.534697 at function evaluation 4
+    Initial fit statistic = 9.178
+    Final fit statistic   = 0.00239806 at function evaluation 4
     Data points           = 3
     Degrees of freedom    = 2
-    Probability [Q-value] = 0.765406
-    Reduced statistic     = 0.267349
-    Change in statistic   = 1.8829
-       const1d.c0     3.39944      +/- 1.74862
+    Probability [Q-value] = 0.998802
+    Reduced statistic     = 0.00119903
+    Change in statistic   = 9.1756
+       exp10.ampl     0.201694     +/- 0.263543
 
 The cache contains 4 elements which we can display::
 
-    >>> print(mdl.c0.val)
-    3.399441714533379
     >>> print(len(mdl._cache))
     4
     >>> for v in mdl._cache.values():
     ...     print(v)
     ...
-    [1. 1. 1.]
-    [1.00034527 1.00034527 1.00034527]
-    [3.39944171 3.39944171 3.39944171]
-    [3.40061543 3.40061543 3.40061543]
-
+    [10.   1.   0.1]
+    [10.00345267  1.00034527  0.10003453]
+    [2.01694277 0.20169428 0.02016943]
+    [2.01763916 0.20176392 0.02017639]
```

Comment:
I was going to ask why you'd changed this, but I see now that `Const1D` is now "cache averse", so you needed a different model.

I do note that I am not happy with the `Exp/Exp10` models - see #340 - but as I wrote that issue 8 years or so ago I guess we are not going to change them now!

Reply:
I did not know about #340 - I think we should change it eventually if we take Sherpa's mission as a general modeling and fitting program seriously [*], but it is definitely not part of this PR.

*: While I don't have hard data to back it up, based on my interaction with users and looking at the occasional tickets we get, it seems that Sherpa is probably used 90% of the time X-ray data with XSPEC models, maybe 7% of the time for images and 3% of the time for other things, so the 1D models in `sherpa.models.basic` don't get exercised a lot. (#713 is another example of that.)